HackAtUCI/irvinehacks-site
# Add daily applications summary charts for admins

Closes #399. The first chart was originally made two years ago for HackUCI 2023. The second chart was newly created for the different types of applicants.

Some examples with sample data:
<img src="https://github.com/user-attachments/assets/7f036642-5ee2-44c6-bf95-2db2edb0dfb2" width="1296" alt="Screenshot of example stacked bar chart" />

<img src="https://github.com/user-attachments/assets/83398e5a-a024-41bc-a04f-8c19bcebd362" width="1296" alt="Screenshot of example stacked area chart" />

Combining the two into one PR since they are closely related, but let me know if it would be easier to review separately.

#### Changes
- Add summary charts to the Applicants page on the Admin site
  - Show daily applications grouped by school as a stacked bar chart
  - Show daily cumulative applications by role as a stacked area chart
- Add `Hacker` to `ParticipantRole` enum

#### Testing
1. Add some different sample applications from different schools
2. Grant yourself a manager role and impersonate the user identity
3. Observe the summary charts appear as expected
